en/vespa8-release-notes.html

```diff
@@ -136,6 +135,17 @@ <h2 id="changed-defaults">Changed defaults</h2>
     </td>
   </tr>
 
+  <tr>
+    <td>
+      Summary features wrapped in <code>rankingExpression</code> have
```

Comment:
"when using an expression as a summary feature, it will no longer be rendered wrapped in rankingExpression()"